components/vocabularies/hpo/api/src/main/java/org/phenotips/vocabulary/internal/solr/HumanPhenotypeOntology.java

```diff
@@ -113,6 +114,15 @@ public String getCitation()
     }
 
     @Override
+    protected Map<String, TermData> load(final String sourceUrl)
```

Comment:
This should instead use the upcoming vocabulary extension paradigm, once it's merged into master.